server/ServerFSM.cpp

```diff
@@ -504,7 +504,7 @@ bool ServerFSM::EstablishPlayer(const PlayerConnectionPtr& player_connection,
         {
             // add "player enter game" message
             boost::posix_time::ptime timestamp = boost::posix_time::second_clock::universal_time();
-            std::string data = "[[PLAYER_ENTERED_GAME," + player_connection->PlayerName() + "]]";
+            std::string data = std::string("[[") + UserStringNop("PLAYER_ENTERED_GAME") + "," + player_connection->PlayerName() + "]]";
```

Comment:
Is the `std::string` wrapping `"[["` neccessary?

Reply:
Yes, without it I get `const char* + const char*` error.